OneCommunityGlobal/HighestGoodNetworkApp
# vitor keep on tab when saving changes

fixes bug 12 from USER PROFILE COMPONENT - (PRIORITY MEDIUM) Jae: When editing a tab on the Profile page, refresh should keep a person on that tab.

## Main changes
1. Now, when the user make changes or hit the save button the data gets updated, but the page doesn't reload, keeping the user in the same tab.

## How to test
1 - Check into current branch
2 - Do npm install and `npm run start:local` to run this PR locally
3 - Log as admin user
4 - Go to dashboard → User Profile → Make changes on every possible changable option, save when needed.(the page should not refresh)
5 - After you manually refresh the page the changes you've made should persirst.

## Example of how to test

https://user-images.githubusercontent.com/17365161/226055648-b74c874f-ca64-4ffe-8be6-bc56ae8618bf.mp4
